photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java

```diff
@@ -132,24 +132,21 @@ public void setExposure(double exposure) {
             // Auto-exposure is active right now, don't set anything.
             return;
         }
+        MathUtils.map(exposure, 0.0, 100.0, 0.1, 100);
 
         // HACKS!
         // If we set exposure too low, libcamera crashes or slows down
         // Very weird and smelly
         // For now, band-aid this by just not setting it lower than the "it breaks" limit
-        //  is different depending on camera.
+        // is different depending on camera.
         if (sensorModel == LibCameraJNI.SensorModel.OV9281) {
-            if (exposure < 6.0) {
-                exposure = 6.0;
-            }
+            exposure = MathUtils.map(exposure, 0.1, 100.0, 6.0, 100);
         } else if (sensorModel == LibCameraJNI.SensorModel.OV5647) {
-            if (exposure < 0.7) {
-                exposure = 0.7;
-            }
+            exposure = MathUtils.map(exposure, 0.1, 100.0, 0.7, 100);
```

Comment:
Why the logic change here?

Reply:
Completely unrelated to grayscale and I should have done this in another PR but this fixes issue #1091 without these changes there is a deadzone on the ov9281 from exposure 0 to 6. Basically it rescales so 6 is the new 0. Same with the ov5647 however with .7 instead of 6. Additionally, since 0 in libcamera also means auto exposure it remaps to from 0 to .1 as the min value as gain and exposure.

Reply:
Are you sure that MathUtils.map will clamp to the min/max?

Reply:
Edit: it does NOT clamp. You'll wanna handle that yourself if that's what you want it to do